css/style.css

```diff
@@ -381,6 +381,14 @@ img {
   gap: 57px;
 }
```

Comment:
`vscode` is still seen in the project directory, but it's not necessary for the project to run. Please add it to `gitignore` file.